terraform/main.tf

```diff
@@ -266,22 +266,16 @@ resource "google_compute_firewall" "allow_mysql" {
   }
 }
 
-# Gets the current version of Kubernetes engine
-data "google_container_engine_versions" "istio_cluster" {
-  zone    = "${var.zone}"
-  project = "${var.istio_project}"
-}
-
 # Creates a GKE cluster to be used in the demo
 resource "google_container_cluster" "istio_cluster" {
   name               = "${var.istio_cluster}"
   zone               = "${var.zone}"
   project            = "${var.istio_project}"
   network            = "${google_compute_network.istio.self_link}"
   subnetwork         = "${google_compute_subnetwork.subnet_istio.self_link}"
-  min_master_version = "${data.google_container_engine_versions.istio_cluster.latest_master_version}"
+  min_master_version = "${var.gke_version}"
 
-  initial_node_count = "3"
+  initial_node_count = "4"
```

Comment:
What size of nodes do we need? We might want to talk a bit about sizing in the README in another PR.

Reply:
Was following the guidance from "Installing istio on GKE"
```
We suggest creating at least a 4 node cluster with the 2 vCPU machine type when using this add-on. You can deploy Istio itself with the default GKE new cluster setup but this may not provide enough resources to explore sample applications
```